apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/SyncOnboardingCompanion.tsx

```diff
@@ -189,15 +192,22 @@ const SyncOnboardingCompanion: React.FC<SyncOnboardingCompanionProps> = ({
           </>
         ),
       },
-      {
-        key: StepKey.Backup,
-        status: "inactive",
-        title: t("syncOnboarding.manual.backup.title"),
-        titleCompleted: t("syncOnboarding.manual.backup.title"),
-        renderBody: () => (
-          <BackupStep device={device} onPressKeepManualBackup={() => setStepKey(StepKey.Apps)} />
-        ),
-      },
+      ...(hasBackupStep
```

Comment:
to avoid spreading an single item array, you can use a conditional object spreading like follow : 

````
...(hasBackupStep
        &&
            {
              key: StepKey.Backup,
              status: "inactive" as StepStatus,
              title: t("syncOnboarding.manual.backup.title"),
              titleCompleted: t("syncOnboarding.manual.backup.title"),
              renderBody: () => (
                <BackupStep
                  device={device}
                  onPressKeepManualBackup={() => setStepKey(StepKey.Apps)}
                />
              ),
            },
          }),

````

Reply:
This does not work as `false` is not spreadable.

Reply:
It wont work outside object but inside object , it will as falsy expression will be treated like empty key object. 
`...(false && {}) `-> won't work </br>
`{...(false && {})} `-> will work

Reply:
But here we are not adding properties into an object, we want to add an object into an **array**, and not add anything into that array if the value is falsy.

Reply:
my bad didn't see the array ! never mind